api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user) who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.
+
+**NOTE**
+Groups are only supported for work or school accounts, and not supported for Microsoft personal accounts.
+
+There are 2 general types of groups, distinguished by their use cases and specific properties' values on each resource. 
+
+| Type              | Use case | groupType | mail-enabled | security-enabled | 
+|-------------------|----------|-----------|--------------|------------------|
+| Office 365 groups | Facilitating user collaboration with shared Microsoft online resources. | ["Unified"] | true | false | 
+| Security groups | Controlling user access to in-app resources. | [] | false | true |
+
+
+## Office 365 groups
+The power of Office 365 groups is in its collaborative nature, perfect for people who work together on a project or a team. They are created with resources that members of the group share including:
+
+- Outlook mail conversations and threads
+- Outlook calendar
+- SharePoint Document Library
+- OneNote notebook
+- SharePoint Team Site
+- Planner
+
+Your app can access and manage these resources through the API.

```

Comment:
I'd like to see some discussion of the difference between Outlook groups and Microsoft Teams -- including the different resources they support. (maybe a table :) ). I think of the journey I had to go on to understand this and the fact that I had internal resources to get me there and I think we are doing our customers a disservice if we don't make this clear.